src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -16,11 +16,11 @@
 import seedu.address.commons.core.index.Index;
 import seedu.address.commons.util.CollectionUtil;
 import seedu.address.logic.commands.exceptions.CommandException;
-import seedu.address.model.person.Address;
```

Comment:
Hmmm why are we changing these package names?

Reply:
It's just to make it more in line with the class structure of "tag", since I'm going to be changing all of addresses, emails and phones to allow for multiple/zero entries